src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -54,6 +56,8 @@
  */
 class CustomerAddressType extends AbstractType
 {
+    private const COMMON_INVALID_CHARS = ' <>;=#{}';
+    private const NAME_INVALID_CHARS = ' 0-9!<>,;?=+()@#"�{}_$%:';
```

Comment:
```suggestion
    private const NAME_INVALID_CHARS = '0-9!<>,;?=+()@#"�{}_$%:';
```
The space shouldn't be here